src/backend/src/prisma/migrations/20221002215248_added_slack_id_to_user_settings/migration.sql

```diff
@@ -0,0 +1,11 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `slackId` on the `User` table. All the data in the column will be lost.
+
+*/
+-- AlterTable
+ALTER TABLE "User" DROP COLUMN "slackId";
+
+-- AlterTable
+ALTER TABLE "User_Settings" ADD COLUMN     "slackId" TEXT;
```

Comment:
this file is deleteing a column from user for some reason. i would delete this migration folder then reset your db then run migrate again

Reply:
this was prob because I originally confused User_Settings with Users but I'll rerun migrate for good measure